Jenkinsfile

```diff
@@ -123,6 +123,22 @@ def deploy(deploy_environment) {
     )
     appImage.push()
   }
+
+  if(deploy_environment == 'production') {
+    cluster_name = 'wifi-api-cluster'
+    service_name = 'authorisation-api-service-wifi'
+    regions = ['eu-west-1', 'eu-west-2']
+  } else {
+    cluster_name = 'staging-api-cluster'
+    service_name = 'authorisation-api-service-staging'
+    regions = ['eu-west-2']
+  }
+
+  regions.each {
+    sh(
+      """aws ecs update-service --force-new-deployment --cluster ${cluster_name} --service ${service_name} --region ${it}"""
```

Comment:
Sadly, putting this on multiple lines is _really_ awkward - some kind of Jenkins-specific Groovy weirdness.